product_docs/docs/pgd/5/manual/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Deploying manually
+navigation:
+- 01-provisioning-hosts
+- 02-install-postgres
+- 03-configuring-repositories
+- 04-installing-software
+- 05-creating-cluster
+- 06-check-cluster
+- 07-configure-proxies
+- 08-using-pgd-cli
+---
+
+The best installation process is automatic, which is why PGD is available and deployable using TPA (Trusted Postgres Architect). 
+Consult [Deploying with TPA](../tpa/) for how to install TPA and use it's automated best-practice driven PGD deployment options for full details.
+Or refer to any of the [Quickstart walkthroughs](../quickstart/) which use TPA to get you up and running quickly.
+
+This section of the documentation looks at the basic operations needed to configure a three-node PGD cluster, with sub-groups, PGD Proxy and PGD CLI.
```

Comment:
_wording suggestion_ "For environments where manual installation and deployment of PGD is desired, EDB provides this section of the documentation, with the information and basic procedures required to manually configure a three-node PGD cluster..."

Reply:
That feels a little like encouraging the user to manually install.